modules/regional/main.tf

```diff
@@ -1,8 +1,10 @@
-
-variable "autospotting_lambda_arn" {}
+module "label" {
+  source  = "git::https://github.com/cloudposse/terraform-null-label.git?ref=0.13.0"
+  context = var.label_context
+}
 
 resource "aws_iam_role" "iam_for_lambda" {
-  name = "iam_for_lambda"
+  name = "${module.label.id}-iam_for_lambda"
```

Comment:
I'm not familiar with this label module, can you share how this ID looks like? Does it contain anything about AutoSpotting so people know this IAM role belongs to it?

Reply:
I grabbed this example from the other module here. I figured following the same pattern made sense.

Reply:
Okay, thanks for the clarification